src/utils/router-config.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { Route, Routes } from "react-router-dom"
+
+const renderRoutes = (routes: RouteType[], extraProps = {}) =>
+  routes ? (
+    <Routes>
+      {routes.map((route) => {
+        if (route.routes) {
+          return (
+            <Route path={route.path} element={route.element}>
```

Comment:
We need to add a `key` for each Route object. Otherwise, there will be a warning in the Chrome console.

![截屏2022-07-26 17 18 30](https://user-images.githubusercontent.com/33231138/180971540-f883e9ce-2a9a-4b6f-b1a5-15864d105a75.png)

Reply:
I've made the required changes.